### PR TITLE
adds three movie view which is displayed upon a 'like'

### PR DESCRIPTION
**Pull request by greysongregory**
allows for 3 like/nope decisions and a done button to dismiss

### CODE REVIEW

**Comment by clusterwin** [comment]
@greysongregory can we see a gif it working?

**Comment by gprasant** [comment]
+1 for gif

File: flixnchill/CardsViewController.m [hunk 1]
```diff
@@ -91,13 +105,84 @@ - (IBAction)onInfoButtonTapped:(UITapGestureRecognizer *)sender {
...
+    [self.view sendSubviewToBack:self.movieCardsView];
+
+}
+
 -(void) initSubViews {
     [self.draggableCard bindWithNextMatch];
 }
 
+
+- (BOOL) fetchMovies {
```
Comment (gprasant) [inline flixnchill/CardsViewController.m:147]: Do you want to use AFNetworking here ? 

```
AFHTTPRequestOperationManager *manager = [AFHTTPRequestOperationManager manager];
[manager GET:@"http://example.com/resources.json" parameters:nil success:^(AFHTTPRequestOperation *operation, id responseObject) {
    NSLog(@"JSON: %@", responseObject);
} failure:^(AFHTTPRequestOperation *operation, NSError *error) {
    NSLog(@"Error: %@", error);
}];
```

File: flixnchill/CardsViewController.m [hunk 2]
```diff
@@ -91,13 +105,84 @@ - (IBAction)onInfoButtonTapped:(UITapGestureRecognizer *)sender {
...
+- (ThreeMoviesView*)setUpMoviesView {
+    ThreeMoviesView *threeMoviesView = [[[NSBundle mainBundle] loadNibNamed:@"ThreeMoviesView" owner:self options:nil] objectAtIndex:0];
+    threeMoviesView.center = CGPointMake(160.0f, 310.0f);
+    threeMoviesView.delegate = self;
+    [threeMoviesView setHidden:YES];
+
+    return threeMoviesView;
+}
+
+- (void)hideBlur {
```
Comment (gprasant) [inline flixnchill/CardsViewController.m]: :+1:  Nice Pattern here. For this couple of methods , Can you make this take in the subView to show / hide ?

File: flixnchill/CardsViewController.m [hunk 3]
```diff
@@ -91,13 +105,84 @@ - (IBAction)onInfoButtonTapped:(UITapGestureRecognizer *)sender {
...
+    [self.view sendSubviewToBack:self.blurView];
+}
+
+- (void)showBlur {
+    [self.blurView setHidden:NO];
+    [self.view bringSubviewToFront:self.blurView];
+}
+
+- (void)onDoneTap {
+    [self.blurView setHidden:YES];
```
Comment (gprasant) [inline flixnchill/CardsViewController.m]: Can you call the hide method above passing in the blurView and moviesView as parameters to a function?

File: ThreeMoviesView.m [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+        [UIView animateWithDuration:.3 animations:^{
+            self.nopeTwo.alpha = 1;
+        }];
+
+    }];
+
+}
+
+- (IBAction)onNopeTapThree:(id)sender {
+    [UIView animateWithDuration:.3 animations:^{
```
Comment (gprasant) [inline ThreeMoviesView.m:125]: :guitar:

File: flixnchill/CardsViewController.m (hunk 1 above)
Comment (greysongregory) [inline flixnchill/CardsViewController.m:147]: done

File: flixnchill/CardsViewController.m (hunk 2 above)
Comment (greysongregory) [inline flixnchill/CardsViewController.m]: good point, done

File: flixnchill/CardsViewController.m (hunk 3 above)
Comment (greysongregory) [inline flixnchill/CardsViewController.m]: yep, done

**Comment by greysongregory** [comment]
Addressed comments. Added gif. Cleaned up some code.

**Comment by greysongregory** [comment]
Merging... feel free to add more comments and I'll address them separately

**Pull request merged**